src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborWriter.String.cs

```diff
@@ -96,11 +113,22 @@ public void WriteEndByteString()
         /// <exception cref="InvalidOperationException">
         ///   Writing a new value exceeds the definite length of the parent data item. -or-
         ///   The major type of the encoded value is not permitted in the parent data item. -or-
-        ///   The written data is not accepted under the current conformance level
+        ///   The written data is not accepted under the current conformance mode
+        /// </exception>
+        public void WriteTextString(string? value) => WriteTextString(value.AsSpan());
```

Comment:
I'd also expect a null string to throw.  CBOR null and CBOR empty string have different major types, after all.

Reply:
Fixed.